build_posix/aclocal/strict.m4

```diff
@@ -138,6 +138,11 @@ AC_DEFUN([AM_CLANG_WARNINGS], [
 	# conditional body.
 	w="$w -Wno-extra-semi-stmt"
 
+	# TODO: Figure out whether we want to disable these or change the code.
```

Comment:
Better putting the follow-on ticket in the comment for easier connection later on.